packages/manager/apps/vrack-services/src/pages/DashboardWrapper.tsx

```diff
@@ -6,31 +6,55 @@ import { useVrackService, useVrackServicesList } from '@/utils/vs-utils';
 import NotFound from '@/pages/404';
 import { ErrorPage } from '@/components/Error';
 import { urls } from '@/router/constants';
+import { DashboardTabItemProps } from '@/components/layout-helpers/layout-helpers.type';
```

Comment:
No it's not possible because they are very different as you can see if you compare them

I would need to add options in the manager-components before doing so but the PR is about tracking here
Probably in the future